driver/main/src/main/java/org/opennms/oce/driver/main/Driver.java

```diff
@@ -118,17 +123,22 @@ public void onSituation(Situation situation) {
                 inventoryDatasource.waitUntilReady();
                 LOG.info("Waiting for alarm datasource...");
                 alarmDatasource.waitUntilReady();
+                LOG.info("Waiting for alarm feedback datasource...");
+                alarmFeedbackDatasource.waitUntilReady();
```

Comment:
Yes it looks like all of these waitUntilReady() calls block until the topic is created. Should I remove the wait for feedback? I think we discussed that the feedback topic won't exist until some feedback has been submitted so that would block the driver on a fresh startup.

Reply:
The call where we do a getAndRegister is also going to block since it makes the same call ultimately. Do we want to add a timeout for how long we are willing to wait on init for the alarmfeedback store to be ready?

Reply:
Added documentation explaining that the driver will be blocked until the topics are present.